example/main.tf

```diff
@@ -0,0 +1,51 @@
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
```

Comment:
add `tags` to all modules and resources (users will be asking about it).
Example https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/examples/complete/main.tf#L67